### PR TITLE
Fixed a couple of bugs for Chef 0.10.0

### PR DESCRIPTION
**Pull request by ekampf**
1. The Knife commands didnt have all the required dependencies loaded
2. The EBS role names did not match their filename and the name provided in the sample clusters

### CODE REVIEW

File: config/client.rb [hunk 1]
```diff
@@ -40,7 +40,8 @@ signing_ca_user      "chef"
 CHEF_CONFIG_FILE      = "/etc/chef/chef-config.json"
 CLIENT_CONFIG_FILE    = "/etc/chef/client-config.json"
 user_data             = OHAI_INFO[:ec2][:userdata]
-chef_config           = JSON.parse(user_data).to_mash rescue {'attributes'=>{}}.to_mash
+#chef_config           = JSON.parse(user_data).to_mash rescue {'attributes'=>{}}.to_mash
+chef_config = {}
```
Comment (howech) [inline config/client.rb:44]: Although I like the idea of eventually getting rid of the user_data stuff, we (Infochimps) still need it at the moment.  Please issue a new pull request without this change.

File: lib/cluster_chef/knife/cluster_kill.rb [hunk 2]
```diff
@@ -158,16 +164,19 @@ class Chef
...
 
 
+
         puts "WARNING!!!!"
         puts
         unless chef_nodes.empty?
           puts "This command will delete the following chef nodes: "
-          puts chef_nodes.map {|n| n.node_name}.join(" ")
+          # puts chef_nodes.map {|n| n.node_name }.join(" ")
+          puts chef_nodes.inspect
```
Comment (howech) [inline lib/cluster_chef/knife/cluster_kill.rb:173]: I want to keep this list brief. When I am killing a cluster of 30+ machines, printing out an inspect for each node makes the display to long to be useful. Please submit a new pull request without this change.

**Comment by howech** [comment]
Nice work! 

Unfortunately, there are a couple of changes mixed in that I don't want to accept - see the notes in the diffs for details.  

Maybe if my git-fu were better, I could just pick the chunks that I want out of the diffs, but I think it would be easier if you would submit a new pull request that does not include the changes that I don't want to incorporate right now. 

Thanks!

**Comment by ekampf** [comment]
The client.rb commit us just something I did today was debugging and wasn't supposed to be part of this pull request (which I made yesterday so I don't know how it got here...)

Will fix as soon as I'm home :)

**Pull request closed without merging**